### PR TITLE
Fixed README.md issue#19

### PR DESCRIPTION
**Pull request by SparshKhanna0001**

### CODE REVIEW

**Comment by SparshKhanna0001** [comment]
@sunny7899 merge it..

File: scraped_data.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[
```
Comment (sunny7899) [inline scraped_data.json:1]: remove these files as they already added to the files folder

File: scraped_data.csv [hunk 2]
```diff
@@ -0,0 +1,10 @@
+content
+"Home
```
Comment (sunny7899) [inline scraped_data.csv:2]: remove these files as they already added to the files folder

**Review by sunny7899 — CHANGES_REQUESTED** [review]

File: scraped_data.csv (hunk 2 above)
Comment (SparshKhanna0001) [reply scraped_data.csv:2]: @sunny7899 which files....?

Comment (sunny7899) [reply scraped_data.csv:2]: @SparshKhanna0001  
image

**Comment by SparshKhanna0001** [comment]
@sunny7899 deleted the files...!

**Comment by sunny7899** [comment]
@SparshKhanna0001 if you got time can you look other PRs there are some conflict in that. That will be great

**Pull request merged**